test/TestUtilities/Test.Utility/Signing/TrustedTestCert.cs

```diff
@@ -73,6 +84,57 @@ private void AddCertificateToStore()
             _store.Add(TrustedCert);
         }
 
+        //According to https://github.com/dotnet/corefx/blob/master/Documentation/architecture/cross-platform-cryptography.md#x509store,
+        //on macOS the X509Store class is a projection of system trust decisions (read-only), user trust decisions (read-only), and user key storage (read-write).
+        //So we have to run command to add certificate to System.keychain to make it trusted.
+        private void AddTrustedCert()
+        {
+            var certFile = new System.IO.FileInfo(System.IO.Path.Combine("/tmp", $"{TrustedCert.Thumbprint}.cer"));
+
+            System.IO.File.WriteAllBytes(certFile.FullName, TrustedCert.RawData);
+
+            string addToKeyChainCmd = $"sudo security add-trusted-cert -d -r trustRoot " +
+                                      $"-k \"/Library/Keychains/System.keychain\" " +
+                                      $"\"{certFile.FullName}\"";
+
+            RunMacCommand(addToKeyChainCmd);
+
+        }
+
+        //According to https://github.com/dotnet/corefx/blob/master/Documentation/architecture/cross-platform-cryptography.md#x509store,
+        //on macOS the X509Store class is a projection of system trust decisions (read-only), user trust decisions (read-only), and user key storage (read-write).
+        //So we have to run command to remove certificate from System.keychain to make it untrusted.
+        private void RemoveTrustedCert()
+        {
+            var certFile = new System.IO.FileInfo(System.IO.Path.Combine("/tmp", $"{TrustedCert.Thumbprint}.cer"));
+
+            string removeFromKeyChainCmd = $"sudo security delete-certificate -Z {TrustedCert.Thumbprint} /Library/Keychains/System.keychain";
```

Comment:
What's the difference between `delete-certificate` and `remove-trusted-cert`?

Also, why not include `-t`?

Reply:
Thanks. Yes, you're right. I should add `-t`
The descriptions of the two are as following,
https://ss64.com/osx/security-delete-cert.html
https://ss64.com/osx/security-cert.html

Reply:
Checked https://ss64.com/osx/security.html, error command.
I run `security error 0`, got `Error: 0x00000000 0 No error`